src/adhocracy_core/docs/authentication_api.rst

```diff
@@ -385,3 +385,67 @@ the user to be logged in from different devices at the same time. ::
     >>> resp_data = testapp.get('/meta_api/', headers=headers).json
     >>> 'resources' in resp_data.keys()
     True
+
+Alternative A: User Password reset without API extension (more backend friendly)
+--------------------------------------------------------------------------------
+
+A password reset link is send by creating a password reset object under the
+``/principals/passwordresets`` pool. The user is identified with the given
+email adress::
+
+    >>> prop = {'content_type': 'adhocracy_core.resources.principal.IPaswordReset',
+    ...         'data': {
+    ...              'adhocracy_core.sheets.principal.IPasswordReset': {
+    ...                  'email': 'anna@example.org'}}}
+    >>> resp_data = testapp.post_json(rest_url + "/principals/passwordresets", prop).json
+
+the url for the created password reset object is protected by obscurity::
```

Comment:
> i think i'm missing a point here. why does that link have to be sent to the user in the first place? shouldn't it be in the email and on the server and nowhere else?

backen sends link to the user, user clicks link, the link is pointing to the fronted, the fronten shows a nice UI and talks with the backend to set the new password